src/repository/storage/postgres/orm/sequelize/note.ts

```diff
@@ -120,7 +120,7 @@ export default class NoteSequelizeStorage {
      */
     this.model.hasOne(this.settingsModel, {
       foreignKey: 'note_id',
-      as: this.settingsModel.tableName,
+      as: 'notesSettings',
```

Comment:
we renamed `notes_settings` to `note_settings`. Where "notesSettings" came from?

Reply:
didn't know that. Renamed to `noteSettings`

Reply:
`noteSettings` is how nested field is called inside `note` model
https://github.com/codex-team/notes.api/pull/85/files#diff-6094ec9c041d26ba04bb33e4cc715c4c70f651ec172802f09ba43aba229aaac0R123
https://github.com/codex-team/notes.api/pull/85/files#diff-790dbfbfc3ab79faef562482e5930737d9b7c352b95e3a0784972d2429b66abfR16